builtin/providers/aws/resource_aws_cloudwatch_log_group.go

```diff
@@ -54,7 +54,7 @@ func resourceAwsCloudWatchLogGroupCreate(d *schema.ResourceData, meta interface{
 		LogGroupName: aws.String(d.Get("name").(string)),
 	})
 	if err != nil {
-		return fmt.Errorf("Creating CloudWatch Log Group failed: %s", err)
+		return fmt.Errorf("Creating CloudWatch Log Group failed: %s:  The CloudWatch Log Group '%s' already exists.", err, d.Get("name").(string))
```

Comment:
I'm afraid we cannot assume here that `ResourceAlreadyExistsException` is the only kind of error we can ever receive during creation, so we'll need to check via `awsErr, ok := err.(awserr.Error); ok && awsErr.Code() == "ResourceAlreadyExistsException"` and only print out the helpful message in such cases.